tests/test_modelutils.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import unittest
+
+import astropy.units as u
+import numpy as np
+
+import pint.models.model_builder as mb
```

Comment:
I don't think this import is needed.

Reply:
It is used in the test, so I'll leave it for now.

Reply:
I see. This is the old method. we can do 
```
from pint.models import get_model
```
And use get_model

Reply:
Please specify which one. All are currently being used in the test...